mabl snippets/ValueFromCommaSeparatedList.js

```diff
@@ -0,0 +1,30 @@
+/**
+ * TITLE: Value from comma separated list
+ * NOTE: This is a 1-indexed list. If you want the first value from the list, set "index" to "1"
+ * 
+ * @mablParam: list - A comma separated list of values that will work like an array
+ * 
+ * @mablParam: index - The index of the item you would like to get from the list
+ * 
+ * @mablReturn: The value from the index in the list
+ * 
+ */
+function mablJavaScriptStep(mablInputs, callback) {
+    // Get the value of the mabl variable "list"
+    let listString = mablInputs.variables.user.list;
+    if (!listString) { console.log("variable 'list' was not created before this Snippet was run.") }
+    // Get the value of the mabl variable "index"
+    let recievedIndex = mablInputs.variables.user.index;
+    let index = recievedIndex ? parseInt(recievedIndex) -1 : 0
```

Comment:
can receivedIndex be 0? 0 - 1 = -1, this index doens't exist. or should we only be providing positive indexes?

Reply:
Added error handling :)